### ISSUE TITLE
Error 0xC00D36B2 when trying to change audio track for live sources

### ISSUE THREAD

**Issue by JezausTevas** [ffmpeginteropx/FFmpegInteropX#395]
Hi!

I am implementing this player into my C# UWP project and have encountered an issue with changing audio tracks.
This occurs when using the provided sample with using default player controls and when modifying `playbackItem.AudioTracks.SelectedIndex` programatically.
The error is very non-descriptive (as usual for windows) and it states:  
> The request is invalid in the current state. (Exception from HRESULT: 0xC00D36B2)

I have tried many different inputs: 
- HLS master playlists
- HLS with multi-track segments
- MPEGTS 
- DASH
All give the same issue as long as they have live content. VOD seems to not be affected.

I have also tried the C++ sample. It also throws error when changing the audio tracks, but this time the error box is empty and the player has text "Error: Video could not be decodec" atop the player controls.

**Comment by brabebhin** [comment]
I don't think we fully support DASH and the like, but if you can provide some sample steps i can look into it.
